spinedb_api/alembic/versions/02581198a2d8_create_entity_class_display_order_tables.py

```diff
@@ -0,0 +1,58 @@
+"""create entity class display order tables
+
+Revision ID: 02581198a2d8
+Revises: ca7a13da8ff6
+Create Date: 2024-07-10 12:13:10.690801
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+from spinedb_api.helpers import DisplayStatus
+
+# revision identifiers, used by Alembic.
+revision = "02581198a2d8"
+down_revision = "c55527151b29"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        "entity_class_display_mode",
+        sa.Column("id", sa.Integer, primary_key=True),
+        sa.Column("name", sa.String(255), nullable=False, unique=True),
+        sa.Column("description", sa.Text(), server_default=sa.null()),
+    )
+    op.create_table(
+        "display_mode__entity_class",
```

Comment:
I think this name breaks the convention with the double underscore. I suggest renaming it to display_mode_entity_class or even entity_class_display_mode. What do you think @jkiviluo @soininen ?

Reply:
Yes, the double underscore does indeed feel inconsistent.

Reply:
There should also be a table display_mode that has 'name' and 'description', so the name should be display_mode__entity_class or entity_class__display_mode. But, yes, we should definitely be consistent.

Reply:
I took the liberty to open an issue about this, see #442.